packages/core/src/app/SettingsMetadata.ts

```diff
@@ -20,6 +20,9 @@ export function createSettingsMetadata() {
       font: new BoolMetaField('legacy font', false).describe(
         "Use the 'JetBrains Mono' font for the user interface.",
       ),
+      coordinates: new BoolMetaField('coordinates', true).describe(
+        'Display coordinates within the preview window.',
```

Comment:
```suggestion
        'Display mouse coordinates within the preview window.',
```

Reply:
Modified the hover message to the suggested.